index.js

```diff
@@ -0,0 +1,81 @@
+const Home = {
+  template: `
+  <div>
+    <h1>Welcome</h1>
+  </div>
+`
+}
+
+const Movies = {
```

Comment:
Convention is to name components with at least two words, to force a more informative naming, such as "MovieList" instead of Movies or "MovieCard" instead of Movie :)